vault/defaults.yaml

```diff
@@ -1,10 +1,9 @@
 vault:
-  version: 0.9.1
+  version: 0.9.6
```

Comment:
Current version is [0.11.2](https://www.vaultproject.io/downloads.html), any reason to bump to `0.9.6`?

Reply:
vault v0.10 introduces kv engine v2 by default, which is not yet supported by salt

Reply:
Ok, I see what you mean.